onnx/serialization.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+__all__ = [
+    "ProtoSerializer",
```

Comment:
nit: Do we need to keep `ProtoSerializer` private at least for now? In the near future possibly ONNX will support more formats and the base code here might change, although it looks good now.

Reply:
SG

Reply:
Done